external/fv3fit/fv3fit/pytorch/graph_loss.py

```diff
@@ -0,0 +1,62 @@
+import torch
+import dataclasses
+
+
+@dataclasses.dataclass
+class LossConfig:
+    """
+    Attributes:
+        loss_type: one of "mse" or "mae"
+    """
+
+    loss_type: str = "mse"
+
+    def __post_init__(self):
+        if self.loss_type not in ("mse", "mae"):
+            raise ValueError(
+                f"loss_type must be 'mse' or 'mae', got '{self.loss_type}'"
+            )
+
+    def loss(self):
+        """
+        Returns the loss function described by the configuration.
+
+        Args:
+            std: standard deviation of the output features
+
+        Returns:
+            loss: pytorch loss function
+        """
+        if self.loss_type == "mse":
+            loss = torch.nn.MSELoss()
+        elif self.loss_type == "mae":
+            loss = torch.nn.L1Loss()
+        else:
+            raise NotImplementedError(f"loss_type {self.loss_type} is not implemented")
+        return loss
+
+    def stepwise_loss(self, multistep, train_model, inputs, labels):
```

Comment:
Should `multistep` be an attribute of this class instead of a method argument? TrainingLoopConfig only uses it to pass to this method.

Reply:
Ironically it's only here because I asked you to move it,  the reason it needs to be moved back is because `stepwise_loss` was moved onto this object.